platforms/ide/ide-native/src/main/java/org/gradle/ide/visualstudio/internal/DefaultVisualStudioProject.java

```diff
@@ -109,17 +105,10 @@ public static String getUUID(File projectFile) {
         return "{" + UUID.nameUUIDFromBytes(projectFile.getAbsolutePath().getBytes()).toString().toUpperCase(Locale.ROOT) + "}";
     }
 
-    @Internal
```

Comment:
This type is part of the mutable model and is no longer used (or should be used) as a @Nested bean.